data/dbConfig.js

```diff
@@ -0,0 +1,4 @@
+const knex = require('knex');
+const knexConfig = require('../knexfile.js');
+
+module.exports = knex(knexConfig.development);
```

Comment:
excellent, cleans up your file structure where you'd have to otherwise require knex just a little bit more